.yamato/metadata.metafile

```diff
@@ -14,24 +14,36 @@ all_platforms:
 
 all_tests:
   - name: integration_tests
-    editors: [2019.4, 2020.3, 2021.2, 2022.1, trunk]
-    args: --type package-tests
+    editors: [2020.3, 2021.2, 2022.1, trunk]
+    args: --type project-tests
+    projects: [HDRP, URP, Standelone]
     variables:
   - name: isolation_tests
     editors: [2020.3, 2021.2, 2022.1, trunk]
     args: --type isolation-tests --enable-load-and-test-isolation
+    projects: [HDRP, URP, Standelone]
```

Comment:
When running isolation tests, a brand new project is created, so I think it doesn't make much sense to supply different projects here

Reply:
Fixed in latest commits